### PR TITLE
Ruby 2.6 is dead

### LINKED ISSUE #28: When ruby 2.6 is retired, allow active support 7+ in gemspec
Ruby 2.6 is EOL the 31/03/2022 (https://endoflife.software/programming-languages/server-side-scripting/ruby)

### PR DESCRIPTION
**Pull request by zaratan**
- For #28: Ruby 2.6 is dead
- For #28: Ruby 2.6 is dead

Closes #28

### CODE REVIEW

**Comment by zaratan** [comment]
@francoisjacques <3

File: shotgrid_api_ruby.gemspec [hunk 1]
```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |spec|
     end
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'activesupport', '< 7' # ActiveSupport 7+ do not support Ruby 2.6
+  spec.add_dependency 'activesupport', '~> 7' # ActiveSupport 7+ do not support Ruby 2.6
```
Comment (francoisjacques) [inline shotgrid_api_ruby.gemspec:36]:
```suggestion
  spec.add_dependency 'activesupport', '~> 7' # ActiveSupport 7+ do not support Ruby 2.6
```

**Review by francoisjacques — APPROVED** [review]
Outdated comment in Gemfile, otherwise it looks good.

**Comment by zaratan** [comment]
@francoisjacques I don't have the permissions to remove "tests (2.6)" from Github CI. Nor merge the PR for that matters.

**Comment by zaratan** [comment]
Tests for Ruby 3.1 should be mandatory too.

**Review by francoisjacques — APPROVED** [review]

**Pull request merged**